docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -95,35 +95,40 @@ spec:
     - nova
 EOF
 ----
++
+[NOTE]
+The service included in the `servicesOverride` key must match the name of the service that you included in the `OpenStackDataPlaneNodeSet` CR. For example, if you use a custom service called `nova-custom`, ensure that you add it to the `servicesOverride` key.
```

Comment:
@gibizer What do you think about this note about the service name in the `servicesOverride` key?

Reply:
I think this note is correct. Both the NodeSet services field and the Deployment sevicesOverride field refers to DataPlaneService CRs by name. The intention here is that even if we use the servicesOverride to run an FFU specific service configuration temporarily, we want that the services list in the NodeSet does not diverge from the services deployed.